src/useRafLoop.ts

```diff
@@ -0,0 +1,39 @@
+import { DependencyList, useEffect, useRef, useState } from 'react';
+
+export type RafLoopReturns = [() => void, boolean | null, () => void];
```

Comment:
I think we can remove `| null` here.

Reply:
Woopsey =)
Left from first implementation. I'll fix it.